lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -201,6 +201,51 @@ describe('ReportRenderer', () => {
       assert.ok(/\dx/.test(descriptionsTxt), 'should have added CPU');
       assert.ok(descriptionsTxt.includes(sampleResults.userAgent), 'user agent populated');
     });
+
+    describe('axe-core', () => {
+      let axe;
+
+      beforeAll(() =>{
+        // Needed by axe-core
+        // https://github.com/dequelabs/axe-core/blob/581c441c/doc/examples/jsdom/test/a11y.js#L24
+        global.window = global.self;
+        global.Node = global.self.Node;
+        global.Element = global.self.Element;
+
+        axe = require('axe-core');
```

Comment:
move to top?

Reply:
It must be required after the global polyfills :(
See: https://github.com/dequelabs/axe-core/blob/581c441c/doc/examples/jsdom/test/a11y.js#L24